subsys/partition_manager/Kconfig

```diff
@@ -75,8 +94,18 @@ endif
 
 endmenu # NCS subsystem configurations
 
+menu "NCS samples configurations"
+
+if NRF_802154_SER_HOST && SOC_NRF5340_CPUAPP
+module=802154_RPMSG
+source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.build_strategy_domain"
```

Comment:
Why not use:
```suggestion
source "$(ZEPHYR_NRF_MODULE_DIR)/subsys/partition_manager/Kconfig.template.build_strategy_domain"
```

Reply:
It does not pass in CI, seems like the checker needs to be updated to support use of MODULE_DIRs.

Reply:
Will check with CI-people

Reply:
Its not supported in CI, there is a rather stale PR here https://github.com/nrfconnect/sdk-zephyr/pull/408 which fixes it. But the rest of this file use the proposed scheme.

Reply:
in that case, they better hurry up and get CI fixed, cause once this gets merged: https://github.com/zephyrproject-rtos/zephyr/pull/30904
all Zephyr modules with glue code outside of the module itself will be using this principle.